app/views/api/v2/webhooks/show.json.rabl

```diff
@@ -4,8 +4,14 @@ object @webhook
 
 extends 'api/v2/webhooks/main'
 
-attributes :target_url, :event, :http_method, :http_content_type, :enabled,
-           :verify_ssl
+attributes :target_url,
+           :event,
+           :http_method,
+           :http_content_type,
+           :enabled,
+           :verify_ssl,
+           # TODO: how do I break these into individual fields?
+           :http_headers
```

Comment:
@ofedoren can you help me how to break up HTTP headers (string JSON) into individual fields? Shall we do this in the API or in the CLI? What's better? Also I need some guidance on how to do this.

Reply:
For the API I'm not sure how to properly do it via rabl. I need to check the documentation.

If there is no way how to convert string into hash object or similar what rabl supports, then I don't mind to do it in the CLI (I guess there would be one field :headers with JSON formatter that should show any custom headers)